### DISCUSSION TITLE
[ideas] Ability to add tags to targets?

### DISCUSSION THREAD

**Discussion by mattmoo** [ropensci/targets#1136] [Ideas]
### Help

- [x] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

I'll start by thanking the team, `targets` has enabled advanced work flows for me that would not have been possible without it.
I'm managing a couple of large pipelines, and one thing that I run into every now and again is how to just work on a small section. It can be things like only making images for a particular report, or only for a particular outcome. Currently I'm doing a mix of separate target lists, patterns in target names, and specific make scripts.
Would it be possible to give targets a 'tag' settable through `tar_target()`? I think that it could help a lot in managing pipelines.

**Comment by wlandau** [comment]
Please see #208. You can use tidyselect selectors and the names argument of tar_make().
